package.json

```diff
@@ -28,53 +43,39 @@
   "homepage": "https://guide.fission.codes",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10.21.0"
+    "node": ">=15"
   },
   "scripts": {
     "docs": "rimraf docs && typedoc && typedoc --entryPoints ./src/fs --out docs/fs",
     "lint": "yarn eslint src/**/*.ts",
-    "prebuild": "rimraf dist && node scripts/gen-version.js",
-    "build": "tsc && rollup -c rollup.config.ts",
-    "start": "tsc -w",
+    "prebuild": "rimraf lib dist && node scripts/gen-version.js",
+    "build": "ttsc && yarn run build:minified && cp src/package.json lib/package.json",
+    "build:minified": "esbuild src/index.ts --outfile=dist/index.min.js --bundle --minify --sourcemap --platform=browser --format=iife --global-name=globalThis.webnative --target=es2020 && gzip -k9 dist/index.min.js",
+    "start": "ttsc -w",
     "test": "jest --forceExit",
```

Comment:
Not totally relevant, but do we want to switch stuff around so `yarn test` works "out of the box"? I would suggest make it the same as `test:unit` and drop `test:unit` ... maybe `test:prod` should be `test:integration` to be more accurate?